client/lib/resize-image-url/index.js

```diff
@@ -25,7 +25,7 @@ const REGEXP_VALID_PROTOCOL = /^https?:$/;
  * @type {number}
  */
 const IMAGE_SCALE_FACTOR =
-	get( typeof window !== 'undefined' && window, 'devicePixelRatio', 1 ) > 1 ? 2 : 1;
+	( typeof window !== 'undefined' && window?.devicePixelRatio ) > 1 ? 2 : 1;
```

Comment:
before the change we always returned `1` but now we might return `false` if `window` is undefined. the result will be the same, but `false > 1` makes me pause to make sure

what about leaving the `> 1` inside the parens?

```js
( typeof window !== 'undefined' && window?.devicePixelRatio > 1) ? 2 : 1;
```

that way the whole subexpression is either `false` or `true` and that feeds into the ternary as a single type instead of as one of two

Reply:
> before the change we always returned `1` but now we might return `false` if `window` is undefined.

I just want to be clear that will definitely not be the case, because of what you say afterwards:

> the result will be the same, but false > 1 makes me pause to make sure

I do agree that's not a great practice, though. Changed to your suggestion (minus the parens, which get removed by `prettier`) 👍

Reply:
> before the change we always returned 1 

poor wording on my part - I meant in the conditional part of the expression.